db/migrate/20170227165349_add_image_file_to_facilities.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageFileToFacilities < ActiveRecord::Migration
+  def change
+    add_column :facilities, :image_file, :string
```

Comment:
I see there is another image field in the same model. Do you think it would be clearer to name this one `icon`?